Libraries/LibWeb/CSS/StyleValues/AbstractImageStyleValue.h

```diff
@@ -48,6 +49,112 @@ enum class GradientRepeating {
     No
 };
 
+enum class GradientSpace {
+    sRGB,
+    sRGBLinear,
+    DisplayP3,
+    A98RGB,
+    ProPhotoRGB,
+    Rec2020,
+    Lab,
+    OKLab,
+    XYZD50,
+    XYZD65,
+    HSL,
+    HWB,
+    LCH,
+    OKLCH,
+};
+
+enum class HueMethod {
```

Comment:
These two enums can be `: u8` which will make your `InterpolationMethod` struct smaller. Not by a whole lot, but it's free.